### DISCUSSION TITLE
user name  and password

### DISCUSSION THREAD

**Discussion by astrobudka** [aaronwmorris/indi-allsky#526] [Q&A] [answered]
Hi there,
when i try to clone, 2nd step... it ask me  for username  and password. What should i wrot there ?

**Comment by bluthen** [comment, marked as answer]
When you try the following?
`git clone https://github.com/aaronwmorris/indi-allsky.git`

No password for me. Are you able to clone other repos through https?

**Reply by astrobudka** [reply]
my bed. :) typo in adress :)
